content/docs/start/experiments/experiment-versioning.md

```diff
@@ -0,0 +1,222 @@
+  code, data, metrics, parameters and plots associated with each experiment,
+  without bloating your Git repo.'
+---
+
+# Get Started: Experiment Versioning
+
+The early stages of a Machine Learning (ML) project are usually exploratory and
+demand fast iterations. Many ML practitioners chose to use tools that require
+little setup and enable rapid prototyping, like those based on
+[Jupyter Notebooks](https://jupyter.org/).
+
+Although these tools are extremely useful, keeping track of the changes you made
+across experiments quickly becomes cumbersome and reproducing an experiment can
+be a challenge.
+
+You can start using DVC to version your experiments without leaving your Jupyter
+Notebook. There are no logins, servers, databases, or UI to spin up. Every
+<abbr>DVC experiment</abbr> will be versioned **without cluttering your repo**,
+unlike saving each run to a separate directory or creating a Git branch for
+each.
+
+## Enable Versioning
+
+All you need to start is a <abbr>DVC repository</abbr> and the [DVCLive] Python
+library installed:
+
+```cli
+$ pip install dvclive
+```
+
+In your Python code, you can start versioning your experiments in DVCLive's
+`Live` API or framework-specific callbacks with `save_dvc_exp=True`.
+
+There are some examples below
+([other frameworks available](/doc/dvclive/api-reference/ml-frameworks)):
+
+<toggle>
+
+<tab title="Pytorch Lightning">
+
+```python
+from dvclive.lightning import DVCLiveLogger
+
+...
+
+trainer = Trainer(logger=DVCLiveLogger(save_dvc_exp=True))
+trainer.fit(model)
+```
+
+</tab>
+
+<tab title="Hugging Face">
+
+```python
+from dvclive.huggingface import DVCLiveCallback
+
+...
+
+trainer.add_callback(DVCLiveCallback(save_dvc_exp=True))
+trainer.train()
+```
+
+</tab>
+
+<tab title="Keras">
+
+```python
+from dvclive.keras import DVCLiveCallback
+
+...
+
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DVCLiveCallback(save_dvc_exp=True)])
+```
+
+</tab>
+
+<tab title="General Python API">
+
+```python
+from dvclive import Live
+
+with Live(save_dvc_exp=True) as live:
+    live.log_param("epochs", NUM_EPOCHS)
+
+    for epoch in range(NUM_EPOCHS):
+        train_model(...)
+        metrics = evaluate_model(...)
+        for metric_name, value in metrics.items():
+            live.log_metric(metric_name, value)
+        live.next_step()
+```
+
+</tab>
+
+</toggle>
+
+After this, each execution of the code will create a <abbr>DVC experiment</abbr>
+containing the results and the changes needed to reproduce it.
+
+[DVCLive] will automatically log some metrics, parameters and plots from the ML
+Framework but you can also manually
+[log additional info](/doc/dvclive/api-reference#log-data) to be included in the
+experiment.
+
+If you have [data tracked by DVC](/doc/start/data-management/data-versioning),
+it will be automatically included in the experiment.
+
+Once you have added [DVCLive] to your code, you can track the changes with git:
```

Comment:
Not sure I get what this sentence is trying to say. Are we trying to make sure the code gets included with the experiments? If so, maybe it needs to be more explicit, like `your code will be included with your experiment if the code file is tracked by git` or something like that.

Reply:
I am removing it for now and revisit it if/when we try to make the example more reproducible or tied to the example repo.